rotkehlchen/db/dbhandler.py

```diff
@@ -583,6 +574,20 @@ def set_settings(self, write_cursor: 'DBCursor', settings: ModifiableDBSettings)
         )
         CachedSettings().update_entries(settings)
 
+    def get_cache(self, cursor: 'DBCursor') -> DBCache:
+        cursor.execute('SELECT name, value FROM key_value_cache;')
+        cache_dict = {}
+        for q in cursor:
+            cache_dict[q[0]] = q[1]
+        return DBCache.deserialize(cache_dict)
```

Comment:
instead of this I believe you can just do

```suggestion
        return DBCache.deserialize(dict(cursor))
```

but check if it works

Reply:
Also this will contain the keys from the updates right? Shouldn't you handle it somehow?